services/QuillLMS/client/app/bundles/Lessons/libs/conceptResults/classroomLessons.js

```diff
@@ -18,6 +18,7 @@ export function generateConceptResult(questionData, studentSubmission) {
       prompt: questionData.prompt,
       answer: studentSubmission.data,
       attemptNumber: 1,
+      question_concept_uid: 'X37oyfiNxSphA34npOb-Ig'
```

Comment:
Might be worth a quick comment explaining why this particular value is hard-coded here.  (I assume that all Lessons have the same concept, so they don't need to pass custom values around, but I could be wrong and other developers might not find it obvious.)